model/tensorflow_hub/dffml_model_tensorflow_hub/text_classifier.py

```diff
@@ -0,0 +1,364 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2019 Intel Corporation
+"""
+Description of what this model does
+"""
+import abc
+import inspect
+from typing import AsyncIterator, Tuple, Any, List, Optional, NamedTuple, Type
```

Comment:
Let's always but `typing` imports at the end of the imports of standard library packages (I've made a note to document this in https://github.com/intel/dffml/issues/339)